_maps/map_files/YogStation/YogStation.dmm

```diff
@@ -25145,6 +24819,28 @@
 /obj/structure/chair/office/dark,
 /turf/open/floor/plasteel/dark,
 /area/maintenance/department/tcoms)
+"fPR" = (
+/obj/effect/turf_decal/bot_red,
+/obj/structure/guncase/shotgun,
+/obj/item/gun/ballistic/shotgun/riot{
+	pixel_x = -3;
+	pixel_y = 3
+	},
+/obj/item/gun/ballistic/shotgun/riot{
+	pixel_x = -3;
+	pixel_y = 3
+	},
+/obj/item/gun/ballistic/shotgun/riot,
+/obj/structure/window/reinforced{
+	dir = 8;
+	max_integrity = 350
+	},
+/obj/machinery/door/window/brigdoor/northright{
+	name = "Armory Door";
+	req_access_txt = "3"
```

Comment:
The windoors appear to all be armory locked
`#define ACCESS_ARMORY 3`

Reply:
Yep we can fix that, we are also adding window shutters as well and removing the window on the right.